src/plugins/authentication/firebase/App.js

```diff
@@ -0,0 +1,58 @@
+import firebase from 'firebase'
+
+export function getLoggedInUser () {
+  const currentUser = firebase.auth().currentUser
+  if (currentUser) {
+    return {
+      email: firebase.auth().currentUser.email,
+      userId: firebase.auth().currentUser.uid,
+      isEmailVerified: firebase.auth().currentUser.emailVerified
+    }
+  } else {
+    return undefined
+  }
+}
+
+export function isAuthenticated () {
+  return (this.getLoggedInUser() && this.getLoggedInUser().isEmailVerified === true) ? true : false
+}
+
+export async function authenticate (email, password) {
+  try {
+    const user = await firebase.auth().signInWithEmailAndPassword(email, password)
+    if (firebase.auth().currentUser.emailVerified === true) {
+      alert(`You are Logged in as ${user.user.email}`)
```

Comment:
alerts are generally a bad idea for web development. during dev its fine. but produciton shouldn't have alerts. usually u give success messages on webpage itself. this is mostly to make it mobile friendly. But also to avoid more interactions